Composer/packages/extensions/visual-designer/src/editors/ObiEditor.tsx

```diff
@@ -424,6 +484,23 @@ export const ObiEditor: FC<ObiEditorProps> = ({
               addCoachMarkRef={addCoachMarkRef}
             />
           </div>
+          <Modal
+            titleAriaId="modalTitle"
+            isOpen={showModal}
+            onDismiss={hideModal}
+            isBlocking={false}
+            containerClassName={contentStyles.container}
+          >
+            <div className={contentStyles.header}>
+              <IconButton
+                styles={iconButtonStyles}
+                iconProps={cancelIcon}
+                ariaLabel="Close popup modal"
+                onClick={hideModal}
+              />
+            </div>
+            <div className={contentStyles.body}>{formatMessage('paste operation not allowed at current element')}</div>
```

Comment:
Alan I think Ben was suggesting we don't show something in the DOM, we send something to the screenreader to announce

Reply:
That is what I meant, yes.

Reply:
This announce message will be added after #2756 merged